### DISCUSSION TITLE
Want to contribute to 1.0.5 to remove bouncy castle and Junit vulnerabilities

### DISCUSSION THREAD

**Discussion by shekharkadam07** [LibrePDF/OpenPDF#933] [General]
Hi,
I wanted to contribute to 1.0.5 to remove bouncy castle and Junit vulnerabilities. Please suggest how do I contribute.

Actually, I have made modifications in pom to overcome these vulnerabilities. 
But I am not able to  push my code to branch due to "Permission Denied" issue see below image. 

![image](https://github.com/LibrePDF/OpenPDF/assets/34413343/c2c1a366-ce90-45e9-b1e0-0b1c4b778912)
 
Please help. 
Thanks!
Chandrashekhar G Kadam

**Comment by andreasrosdalw** [comment]
There are no Bouncy castle or Junit vulnerabilities. Please elaborate.

**Reply by asturio** [reply]
And just to add my 2 cents to this. Everyone with a GitHub-account can contribute to OpenPDF, there is no need to you to be added as a "official OpenPDF (member|contributor|maintainer)". Just:

1. Fork OpenPDF in GitHub
2. Make your changes in your fork
3. When you are done, just create a pull request, for your changes to be merged in OpenPDF. If they are OK, it will be merged.
